airflow/dags/ingestion/ask-astro-load-stackoverflow.py

```diff
@@ -0,0 +1,89 @@
+import os
+from datetime import datetime
+
+from include.tasks import extract, ingest, split
+from weaviate_provider.operators.weaviate import WeaviateCheckSchemaBranchOperator
+
+from airflow.decorators import dag, task
+from airflow.providers.slack.operators.slack import SlackAPIPostOperator
+
+try:
+    ask_astro_env = os.environ["ASK_ASTRO_ENV"]
+except Exception:
+    ask_astro_env = "NOOP"
+
+if ask_astro_env == "prod":
+    _WEAVIATE_CONN_ID = "weaviate_prod"
+    _SLACK_CONN_ID = "slack_api_ro"
+    _GITHUB_CONN_ID = "github_ro"
+elif ask_astro_env == "local":
+    _WEAVIATE_CONN_ID = "weaviate_local"
+    _SLACK_CONN_ID = "slack_api_ro"
+    _GITHUB_CONN_ID = "github_ro"
+elif ask_astro_env == "dev":
+    _WEAVIATE_CONN_ID = "weaviate_dev"
+    _SLACK_CONN_ID = "slack_api_ro"
+    _GITHUB_CONN_ID = "github_ro"
+elif ask_astro_env == "test":
+    _WEAVIATE_CONN_ID = "weaviate_test"
+    _SLACK_CONN_ID = "slack_api_ro"
+    _GITHUB_CONN_ID = "github_ro"
```

Comment:
Look like we can just use `f"weaviate_{ask_astro_env}"`? other looks like the same